sovrin_client/test/cli/test_send_node_wrong_node_ip.py

```diff
@@ -0,0 +1,32 @@
+import pytest
```

Comment:
In order to do this we have to clone a number of existing fixtures to change their scope from module to function because now (at least before Sasha's changes) some invalid parameters of `send NODE` command turn the pool into unrecoverable state.
So is it ok to clone the fixtures with only scope changed?